lib/prepare_release.js

```diff
@@ -249,6 +268,14 @@ class ReleasePreparation {
     ]).trim();
   }
 
+  async getLTSCodename(version) {
+    const { cli } = this;
+    return await cli.prompt(
+      'Enter the LTS code name for this release line:',
+      { questionType: 'input', noSeparator: true, defaultAnswer: '' }
+    );
```

Comment:
Add a link to https://github.com/nodejs/Release/blob/master/CODENAMES.md ?

Reply:
> 
> 
> Add a link to https://github.com/nodejs/Release/blob/master/CODENAMES.md ?

Done.